components/organisms/ChangePasswordUser/ChangePasswordUser.tsx

```diff
@@ -107,14 +68,37 @@ export const ChangePasswordUser = () => {
       );
     }
   };
+  const validatePasswordUser = async () => {
+    if (userInfo?.uid !== undefined) {
+      const validatePasswordData = await validatePassword(
```

Comment:
This variable should be `passwordValidation` or `isValidatedPassword`

Reply:
changed This variable should be validateUserPassword

Reply:
changed passwordValidation